s2p/__init__.py

```diff
@@ -588,17 +605,25 @@ def main(user_cfg, start_from=0):
         print('3) rectifying tiles...')
         parallel.launch_calls(rectification_pair, tiles_pairs, nb_workers,
                               timeout=timeout)
-        tiles = [t for t in tiles if t is not None]
```

Comment:
Will removing this cause the missing tiles crash again?

Reply:
no I think this wasn't actually being used, since I wasn't using returned tiles from `rectification_pair`